### PR TITLE
Article list

### PR DESCRIPTION
**Pull request by manggalang**

### CODE REVIEW

File: app/assets/stylesheets/pages/articles/article-list/_article-list.scss [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+			font-size: 33px;
+		}
+
+		@media only screen and (max-width: 640px){
+			font-size: 31px;
+		}
+	}
+
+	&__item{
+		display: block;
```
Comment (Fajaru1503) [inline app/assets/stylesheets/pages/articles/article-list/_article-list.scss]: it's useless, remove it

File: app/assets/stylesheets/pages/articles/article-list/_article-list.scss [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+		}
+
+		@media only screen and (max-width: 640px){
+			font-size: 14px;
+		}
+	}
+
+	&__author{
+		color: $textColorSecondary;
+		font-size: 16px;
```
Comment (Fajaru1503) [inline app/assets/stylesheets/pages/articles/article-list/_article-list.scss]: it's the default size, so kind of useless if you describe it, remove it

File: app/views/client/articles/_article-list.html.erb [hunk 3]
```diff
@@ -0,0 +1,66 @@
+<div class="article-list">
+	<div class="container-large">
+		<h2 class="article-list__title">intermezzo</h2>
+			
+		<div class="article-list__item">
+			<a href="#" class="article-list__item-link">
```
Comment (Fajaru1503) [inline app/views/client/articles/_article-list.html.erb]: tag a nya dikasi class "article-list__item", lalu container "article-list__item" diatas dihapus, dan properti dari class "article-list __item-link" dipindah

File: app/assets/stylesheets/pages/articles/article-list/_article-list.scss [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+	}
+
+	&__topics{
+		margin-top: 2px;
+		background-color: $colorPrimary;
+		color: #fff;
+		text-transform: uppercase;
+		font-size: 14px;
+		padding: 5px 8px;
+		width: 106px;
```
Comment (Fajaru1503) [inline app/assets/stylesheets/pages/articles/article-list/_article-list.scss]: jebol, coba diperhatikan lagi, sesuaikan dengan mockup

File: app/assets/stylesheets/pages/articles/article-list/_article-list.scss (hunk 1 above)
Comment (manggalang) [reply app/assets/stylesheets/pages/articles/article-list/_article-list.scss]: okay

File: app/assets/stylesheets/pages/articles/article-list/_article-list.scss (hunk 2 above)
Comment (manggalang) [reply app/assets/stylesheets/pages/articles/article-list/_article-list.scss]: okok

File: app/views/client/articles/_article-list.html.erb (hunk 3 above)
Comment (manggalang) [reply app/views/client/articles/_article-list.html.erb]: already change class of tag a and delete class item-link

File: app/assets/stylesheets/pages/articles/article-list/_article-list.scss (hunk 4 above)
Comment (manggalang) [reply app/assets/stylesheets/pages/articles/article-list/_article-list.scss]: okay

**Pull request merged**